app/master/build_preparer.py

```diff
@@ -9,7 +9,7 @@
 from app.project_type.project_type import ProjectType
 
 
-class SerialRequestHandler(object):
+class BuildPreparer(object):
```

Comment:
We could just move all this logic entirely into the `Build` class. It seems like most of the changes to this class since the beginning have been to move stuff out of here into `Build`, so it feels like the direction we've been moving for awhile.

I totally agree with you on the "verby" class names -- I feel like it's kind of a signal that the code is structured more imperatively than object-oriented.